packages/components/src/components/typography/Heading/Heading.tsx

```diff
@@ -27,36 +28,28 @@ const baseStyles = css`
     line-height: normal;
     letter-spacing: normal;
     ${(props: HeadingProps) => props.fontWeight && fontWeightStyle}
-    color: ${({ theme }) => theme.TYPE_DARK_GREY};
+    color: ${({ theme }) => theme.textDefault};
```

Comment:
this is redundant as well, I believe. Should already be in the global style somewhere

Reply:
text-rendering and font- smoothing should all be in [GlobalStyles](https://github.com/trezor/trezor-suite/blob/7d23220dc5079f2433feb107fe9efac2cfad2ff9/packages/suite/src/support/suite/styles/GlobalStyle.tsx#L21), so feel free to remove

Reply:
I think everything except for `text-align` could be removed. UPD: just checked - it's not used anywhere – could be removed completely.